i18n/zh/docusaurus-plugin-content-docs/current/upgrade/v1-1-to-v1-1-2.md

```diff
@@ -4,7 +4,11 @@ sidebar_label: Upgrade from v1.1.0/v1.1.1 to v1.1.2
 title: "Upgrade from v1.1.0/v1.1.1 to v1.1.2"
 ---
 
+:::danger
 
+Please do not upgrade a running cluster to v1.1.2 if your machine has an **Intel E810** NIC card. We saw some reports that the NIC card has a problem when added to a bonding device. Please check this issue for more info: https://github.com/harvester/harvester/issues/3860.
```

Comment:
```suggestion
如果您的机器有 **Intel E810** NIC 卡，请不要将正在运行的集群升级到 v1.1.2。 我们看到一些报告说将网卡在添加到Bond设备时会出现问题。 请检查此问题以获取更多信息：https://github.com/harvester/harvester/issues/3860。
```

Reply:
Slightly updated the translation on top of your suggestions (see the edit history of this line to see what is changed).